kitsune/sumo/management/commands/manage_l10n.py

```diff
@@ -45,6 +45,8 @@ def handle(self, *args, **options):
                     "locale/templates/LC_MESSAGES/django.pot",
                     "-k",
                     "_lazy",
+                    "-k",
+                    "pgettext_lazy",
```

Comment:
Some of the Python files also use `pgettext_lazy`. I'm not sure if it's ever used within JS files.